tox.ini

```diff
@@ -35,14 +35,13 @@ commands=
     rpc-state-quadratic: pytest {posargs:tests/trinity/json-fixtures-over-rpc/test_rpc_fixtures.py -k 'GeneralStateTests and stQuadraticComplexityTest'}
     transactions: pytest {posargs:tests/json-fixtures/test_transactions.py}
     vm: pytest {posargs:tests/json-fixtures/test_virtual_machine.py}
-    native-blockchain: pytest {posargs:tests/json-fixtures/test_blockchain.py}
```

Comment:
> Good question. You mean like this one?..
> 
> ```
> pytest -k 'test_blockchain_fixtures[/home/veox/src/py-evm/fixtures/BlockchainTests/bcUncleTest/uncleWithSameBlockNumber.json:uncleWithSameBlockNumber_Homestead]'
> ```
> They are in the same submodule that gets pulled in as `fixtures`.
> 
> (I haven't tested this particular PR yet, and whether it has inadvertently removed something.)

@veox right, the complete removal of this test (non-state blockchain tests) is what threw me off. Just wanted to make sure we had confirmation that nothing got dropped in the refactor.

Reply:
The only way we would be missing some tests would be if this collection of forks doesn't cover the full set of blockchain tests: `frontier, homestead, eip150, eip158, byzantium, constantinople, metropolis`. Let me think of a way to test whether that's true.

Reply:
Well shucks, looks like we're missing some tests:

```
(py-evm-ojdIf4Yr) brian@rhythm:~/ef/py-evm$ pytest tests/json-fixtures/test_blockchain.py --collect-only 2>/dev/null | grep -oP '\[\K.*(?=\])' | cut -d':' -f3 | sort | uniq
Byzantium
Constantinople
EIP150
EIP158
EIP158ToByzantiumAt5
Frontier
FrontierToHomesteadAt5
Homestead
HomesteadToDaoAt5
HomesteadToEIP150At5
```

Reply:
Okay, thanks for catching this Carver! I've pushed [a commit](https://github.com/ethereum/py-evm/pull/1577/commits/9d28e837fb7722ab46858342fbd22024cf088cc1) which adds the missing tests.

Reply:
Haha, well I pretty much just pointed and grumbled. Thanks for hunting it down!